ui/stories/http-requests-bar-chart.stories.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/extensions */
+import hbs from 'htmlbars-inline-precompile';
+import { storiesOf } from '@storybook/ember';
+import { withKnobs, object } from '@storybook/addon-knobs';
+import notes from './http-requests-bar-chart.md';
+
+const COUNTERS = [
+  { start_time: '2019-04-01T00:00:00.000Z', total: 5500 },
+  { start_time: '2019-05-01T00:00:00.000Z', total: 4500 },
+  { start_time: '2019-06-01T00:00:00.000Z', total: 5000 },
+];
+
+storiesOf('HttpRequests/BarChart/', module)
+  .addParameters({ options: { showPanel: true } })
+  .addDecorator(
+    withKnobs()
+  )
+  .add(`HttpRequestsBarChart`, () => ({
+    template: hbs`
+        <h5 class="title is-5">Http Requests Bar Chart</h5>
+        <HttpRequestsBarChart @counters={{counters}}/>
+    `,
+    context: {
+      counters: object('counters', COUNTERS)
+    },
+  }),
+  {notes}
+);
```

Comment:
I think it's worth adding a story for a no data bar chart. Which I also noticed isn't being handled yet. 

No pressure to implement this now if this is time-sensitive, since I imagine the metrics API won't ever return an empty response, but it would still be nice to have that base covered with the chart.

Reply:
Yeah, I specifically didn't add this case because the bar chart will never be rendered if there is no data. If we refactor this bar chart to be more generic down the line I'll definitely add an example with no data.